QueryBuilder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace CanalTP\NavitiaPhp;
+
+class QueryBuilder
+{
+    /**
+     * @var string
+     */
+    private $coverage = null;
+
+    /**
+     * @var string
+     */
+    private $api = null;
+
+    /**
+     * @param string $coverage
+     *
+     * @return self
+     */
+    public function coverage($coverage)
+    {
+        $this->coverage = $coverage;
+
+        return $this;
+    }
+
+    /**
+     * @param string $api
+     *
+     * @return self
+     */
+    public function api($api)
```

Comment:
This only work for what you cover which is simple call, but don't cover two level api like : https://api.navitia.io/v1/coverage/fr-idf/coords/2.377310;48.847002/pois?distance=1000